src/function/algebra/simplify.js

```diff
@@ -425,7 +425,9 @@ export const createSimplify = /* #__PURE__ */ factory(name, dependencies, (
       assuming: { multiply: { associative: true } }
     },
 
-    { l: 'n1/(-n2)', r: '-n1/n2' }
+    { l: 'n1/(-n2)', r: '-n1/n2' },
+
+    { l: 'n+-(n1)', r: 'n-(n1)' }
```

Comment:
😎  so simple. 

I somehow expected this case to be handled by the rule `{ l: 'n+-n1', r: 'n-n1' }` at line 394. Can you put this new rule right under line 394 to keep these two similar rules grouped together?

Reply:
sure

Reply:
👍